server/migrations/20190111130634-create-reply.js

```diff
@@ -0,0 +1,33 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Replies', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    userId: {
+      type: Sequelize.INTEGER,
+      allowNull: false
+    },
```

Comment:
I noticed that the Comment model has a column for content but the Reply model doesn't have

Reply:
It's been added now thanks.